bitcoin/bitcoind/wallet.go

```diff
@@ -92,13 +96,22 @@ func (w *BitcoindWallet) Start() {
 	if w.trustedPeer != "" {
 		args = append(args, "-connect="+w.trustedPeer)
 	}
+	if w.userTor {
+		var port int
+		if w.controlPort == 9151 {
```

Comment:
Q: what is the port number adjustment for?
Q2: is it ok to do this after what happens online 90
`args := []string{"-walletnotify='" + path.Join(w.repoPath, "notify.sh") + " %s'", "-server", "-torcontrol=127.0.0.1:" + strconv.Itoa(w.controlPort)}`